pycode/examples/secir_groups.py

```diff
@@ -52,55 +54,54 @@ def run_secir_groups_simulation():
 
         # Initial number of peaople in each compartment
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Exposed)] = 100
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Carrier)] = 40
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Infected)] = 80
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Hospitalized)] = 40
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.ICU)] = 20
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Recovered)] = 7
-        model.populations[secir.AgeGroup(i), secir.Index_InfectionState(secir.InfectionState.Dead)] = 3
-        model.populations.set_difference_from_group_total_AgeGroup((secir.AgeGroup(i), 
-            secir.Index_InfectionState(secir.InfectionState.Susceptible)), populations[i])
+        model.populations[AgeGroup(i), Index_InfectionState(State.Exposed)] = 100
+        model.populations[AgeGroup(i), Index_InfectionState(State.Carrier)] = 50
+        model.populations[AgeGroup(i), Index_InfectionState(State.Infected)] = 50
+        model.populations[AgeGroup(i), Index_InfectionState(State.Hospitalized)] = 20
+        model.populations[AgeGroup(i), Index_InfectionState(State.ICU)] = 10
+        model.populations[AgeGroup(i), Index_InfectionState(State.Recovered)] = 10
+        model.populations[AgeGroup(i), Index_InfectionState(State.Dead)] = 0
+        model.populations.set_difference_from_total((AgeGroup(i), Index_InfectionState(State.Susceptible)), populations[i])
 
         # Compartment transition propabilities
-        model.parameters.probabilities[i].set_infection_from_contact(1.0)
-        model.parameters.probabilities[i].set_carrier_infectability(0.67)
-        model.parameters.probabilities[i].set_asymp_per_infectious(0.09)
-        model.parameters.probabilities[i].set_risk_from_symptomatic(0.25)
-        model.parameters.probabilities[i].set_hospitalized_per_infectious(0.2)
-        model.parameters.probabilities[i].set_icu_per_hospitalized(0.25)
-        model.parameters.probabilities[i].set_dead_per_icu(0.3)
-        
+           
+        model.parameters.RelativeCarrierInfectability[AgeGroup(i)] = 0.67  #TODO: must this parameter be set? 
+        model.parameters.InfectionProbabilityFromContact[AgeGroup(i)] = 1.0
+        model.parameters.AsymptoticCasesPerInfectious[AgeGroup(i)] = 0.09  # 0.01-0.16
+        model.parameters.RiskOfInfectionFromSympomatic[AgeGroup(i)] = 0.25  # 0.05-0.5
+        model.parameters.HospitalizedCasesPerInfectious[AgeGroup(i)] = 0.2  # 0.1-0.35
+        model.parameters.ICUCasesPerHospitalized[AgeGroup(i)] = 0.25  # 0.15-0.4
+        model.parameters.DeathsPerHospitalized[AgeGroup(i)] = 0.3  # 0.15-0.77
+        #TODO what about MaxRiskOfInfectionFromSympomatic?
```

Comment:
Please just set it as a dummy to twice the RiskOfInfectionFromSymptomatic. It is only used if the test-and-trace-capacity is used. You can add this as a comment. Good examples on the parameters that can be used should be get from the cpp tests: https://github.com/lelange/memilio/blob/main/cpp/tests/test_secir.cpp and https://github.com/lelange/memilio/blob/main/cpp/tests/test_secir_ageres.cpp